ground/gcs/src/plugins/gpsdisplay/gpssnrwidget.cpp

```diff
@@ -101,7 +101,7 @@ void GpsSnrWidget::drawSat(int drawIndex, int index)
     const int prn = satellites[index][0];
     const int snr = satellites[index][3];
     if (prn && snr) {
-        boxes[index]->show();
+        boxes[drawIndex]->show();
```

Comment:
Bad choice of name originally led to this, they should have added more to both symbol names.